app/features-json/setup_json_controller.rb

```diff
@@ -0,0 +1,12 @@
+require_relative "api_controller"
+
+module FastlaneCI
+  # Controller for providing all setup APIs
+  class SetupJSONController < APIController
+    HOME = "/data/setup"
+
+    get "#{HOME}/configured", authenticate: false do
+      return json(Services.onboarding_service.correct_setup?)
```

Comment:
it's just a boolean, so i'm not sure we actually need the `json()` method. Please advise.

Reply:
Let's actually return a real object here, maybe a `ConfigurationData`. That way we can add more configuration properties as needed in the future with out having to refactor.

Reply:
Also, let's call this endpoint `"#{HOME}/configuration"`

Reply:
Does that come with any privacy/security side effects, if a user has only e.g. very basic read access to some repos? Either way, this is not a blocker - let's merge this PR as Nima is OOO right now